client/src/imgDataS3/index.js

```diff
@@ -0,0 +1,118 @@
+const imgData = [
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo1.png',
+    title: 'tattoo1',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo2.png',
+    title: 'tattoo2',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo3.png',
+    title: 'tattoo3',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo4.png',
+    title: 'tattoo4',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo5.png',
+    title: 'tattoo5',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo6.png',
+    title: 'tattoo6',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo7.png',
+    title: 'tattoo7',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo8.png',
+    title: 'tattoo8',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo9.png',
+    title: 'tattoo9',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo10.png',
+    title: 'tattoo10',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo11.png',
+    title: 'tattoo11',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo12.png',
+    title: 'tattoo12',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo13.png',
+    title: 'tattoo13',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo14.png',
+    title: 'tattoo14',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo15.png',
+    title: 'tattoo15',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo16.png',
```

Comment:
i see you stored these in s3, you can delete the local ones

Reply:
Done. There is still the empty folder. I'll get rid of that once this is merged. Thanks!